src/docs/platforms/php/laravel.mdx

```diff
@@ -215,8 +215,7 @@ $ php artisan sentry:test
 ### Local development
 
 When Sentry is installed in your application it will also be active when you are developing.
-
-If you don't want errors to be sent to Sentry when you are developing set the DSN value to `null`.
+If you want errors to be sent to Sentry when you are developing set the DSN value to `null`.
```

Comment:
Oooops, I think this was a bad change?

Reply:
https://github.com/getsentry/sentry-docs/pull/1974